server/members/migrations/0004_user_members_use_usernam_82d39a_idx_and_more.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 4.2.16 on 2024-11-30 23:07
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('members', '0003_user_profile_picture_url'),
+    ]
+
+    operations = [
+        migrations.AddIndex(
+            model_name='user',
+            index=models.Index(fields=['username'], name='members_use_usernam_82d39a_idx'),
+        ),
+        migrations.AddIndex(
+            model_name='user',
+            index=models.Index(fields=['first_name'], name='members_use_first_n_056ccc_idx'),
+        ),
+        migrations.AddIndex(
+            model_name='user',
+            index=models.Index(fields=['last_name'], name='members_use_last_na_5b1ab6_idx'),
+        ),
+        migrations.AddIndex(
```

Comment:
not exactly, it is a composite index on the two columns. probably not a must-have since we could theoretically just index on the concat of the two for most searches, and also cardinality of duplicate names is relatively low.